environments/gcp/provider.tf

```diff
@@ -0,0 +1,31 @@
+provider "google" {
+  project = "flyte-terraform1-403721" #Replace with your project name
+  region  = "us-east1"
+}
```

Comment:
I would use the values the user adapted in `locals.tf`. This way, the user one has to modify a single file.